.gitpod.yml

```diff
@@ -10,10 +10,10 @@ tasks:
     prebuild: |
       sudo docker-up &
       export DDEV_NONINTERACTIVE=true
-      .ddev/gitpod-setup-ddev.sh
+      .gitpod/gitpod-setup-ddev.sh
```

Comment:
This is a good idea but it will break ddev (which is OK).

But it also has a line in it where it figures out what directory *it* lives in, so this will break the use of this script until you change that.

Reply:
https://github.com/shaal/ddev-gitpod/blob/c759573c6feec8079190ed09a564b10fc57ab30f/.ddev/gitpod-setup-ddev.sh#L7